iam-ape/iam_ape/evaluator.py

```diff
@@ -628,7 +628,7 @@ def get_direct_policies(
     ) -> List[PolicyWithSource]:
         inline_policies = [
             PolicyWithSource(
-                "inline policy", normalize_policy(policy["PolicyDocument"])
+                policy["PolicyName"], normalize_policy(policy["PolicyDocument"])
```

Comment:
Inline policies aren't always guaranteed to have a `PolicyName`, better use 
```python
policy.get("PolicyName", "inline policy")
```

```suggestion
                policy.get("PolicyName", "inline policy"), normalize_policy(policy["PolicyDocument"])
```